code/game/objects/structures/misc_structures.dm

```diff
@@ -0,0 +1,60 @@
+/obj/structure/headpole
+	name = "pole"
+	icon = 'icons/obj/structures.dmi'
+	icon_state = "pole"
+	desc = "How did this get here?"
+	density = 0
+	anchored = 1
+	var/obj/item/weapon/spear/spear = null
+	var/obj/item/weapon/organ/head/head = null
+	var/obj/item/weapon/organ/head/display_head = null
+
+/obj/structure/headpole/New(atom/A, var/obj/item/weapon/organ/head/H)
+	..(A)
+	if(istype(H))
+		head = H
+		name = "[H.name]"
+		if(H.origin_body)
+			desc = "The severed head of [H.origin_body.real_name], crudely shoved onto the tip of a spear."
+		else
+			desc = "A severed head, crudely shoved onto the tip of a spear."
+		display_head = new (src)
```

Comment:
It should also be possible to make an object (or an image), set its appearance var to that of the head, and then manually reset any of the vars you don't want, rather than setting most of them yourself.